hardhat.config.ts

```diff
@@ -89,7 +89,7 @@ const config: HardhatUserConfig = {
         settings: {
           optimizer: {
             enabled: true,
-            runs: 10000,
+            runs: 200,
```

Comment:
Quite the difference!?!

Reply:
Thanks for commenting on this, I went back and realized I misunderstood this and had it backwards